src/app/test_executive/snapps_test.ml

```diff
@@ -18,8 +17,19 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     let open Test_config in
     let open Test_config.Block_producer in
     { default with
-      requires_graphql = true
-    ; block_producers = [ { balance = "4000000000"; timing = Untimed } ]
+      requires_graphql =
+        true
+        (* must have at least as many block producers as party's in any
+           Parties.t, because we need that many keypairs
+        *)
+    ; block_producers =
+        [ { balance = "3000000000"; timing = Untimed }
+        ; { balance = "3000000000"; timing = Untimed }
+        ; { balance = "3000000000"; timing = Untimed }
+        ; { balance = "3000000000"; timing = Untimed }
+        ; { balance = "3000000000"; timing = Untimed }
+        ; { balance = "3000000000"; timing = Untimed }
+        ]
```

Comment:
Can you add another field here, say `transaction_accounts`
```
transaction_accounts =
[ { keypair =...; balance= ...; timing=...}...]
```

You can add how many ever accounts you'd like to see in the ledger (I think there's a hard limit on the size of the config file because of some terraform thing but 10-20 accounts should be fine) without having to spin up that many block producers.

Then in mina_automation.ml you can add these new accounts to `runtime_accounts` here 
https://github.com/MinaProtocol/mina/blob/733f61a0c2c33b689988638359cfb51b03932e5c/src/lib/integration_test_cloud_engine/mina_automation.ml#L106

using the function https://github.com/MinaProtocol/mina/blob/733f61a0c2c33b689988638359cfb51b03932e5c/src/lib/integration_test_cloud_engine/mina_automation.ml#L114

These accounts then get added to the ledger

Reply:
Will do this in a separate PR, if it looks feasible.